src/cockpit/packages.py

```diff
@@ -229,7 +229,8 @@ def add_file(self, item: Path, checksums: List['hashlib._Hash']) -> None:
             lower_locale = locale.lower().replace('_', '-')
             self.translations[lower_locale] = data, guessed_type
         else:
-            self.files[rel.removesuffix('.gz')] = data, guessed_type
+            index = rel[:-3] if rel.endswith('.gz') else rel
```

Comment:
`index` is a very confusing name.  I'd expect that to be an int used to subscript the string, but it's the resulting string to be used as the key.  Maybe call it `basename` or `key` instead?

Reply:
"basename" sounds good to me, changed.